lib/blueprinter/configuration.rb

```diff
@@ -1,10 +1,11 @@
 module Blueprinter
   class Configuration
-    attr_accessor :generator, :method
+    attr_accessor :generator, :method, :sort_by_definition
 
     def initialize
       @generator = JSON
       @method = :generate
+      @sort_by_definition = false
```

Comment:
just figured a boolean comparison would be easier and lighter for execution [here](https://github.com/procore/blueprinter/pull/119/files#diff-d909c0537c86c6300c32c53da4d2c8c5R58). Going by that context, I couldn't think of a more apt variable name. If you feel strongly otherwise, I could go ahead with your approach as well.

Reply:
I like @mcclayton suggestion too!